lib/zone.ts

```diff
@@ -648,6 +651,17 @@ const Zone: ZoneType = (function(global: any) {
       return _currentTask;
     };
 
+    static __load_patch(name: string, fn: _PatchFn): void {
```

Comment:
should we allow 3rd party add their own patch?
 If so, we may need to consider which Zone Private API should we expose.